### ISSUE TITLE
Improve the `current_cases` fixture: add the full case id

### ISSUE THREAD

**Issue by smarie** [smarie/python-pytest-cases#217]
With [the `current_cases` fixture](https://smarie.github.io/python-pytest-cases/#c-accessing-the-current-case) one can access all information about the case currently being used for a given parameter.

However if the case is itself parametrized, the id available in `current_cases[argname].id` only corresponds to the "base" of the case, and is therefore not unique. This is a problem if we wish to use this id to fill a results table in a benchmark for example.

Proposal: add a `.full_id` argument that should return the part of the pytest id that corresponds to this case, therefore including the parameters and also the required fixture parameters (see #216).

Basically an id representing the current callspec of the case.
